src/components/WithTrackingHOC/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react'
+import ReactGA from 'react-ga'
+
+interface WithTrackerHOCProps extends Window {}
+
+export default function withTracker(WrappedComponent: React.SFC | React.ComponentClass, options = {}) {
+  const trackPage = (page: string) => {
+    ReactGA.set({
+      page,
+      ...options,
+    })
+    ReactGA.pageview(page)
+  }
+
+  return class extends Component<WithTrackerHOCProps> {
+    componentDidMount() {
+      const page = this.props.location.pathname + this.props.location.hash
+      trackPage(page)
+    }
+
+    componentWillReceiveProps(nextProps: any) {
+      const currentPage = this.props.location.pathname + this.props.location.hash
+      const nextPage = nextProps.location.pathname + nextProps.location.hash
+
+      if (currentPage !== nextPage) {
```

Comment:
This would trigger when user goes between different hashes inside `/content/Tokens` path.
React-router already correctly parses the url like `/#/content/Tokens#what-is-mgn` into pathname `"/content/Tokens"` and hash thanks to HashHistory

Reply:
not sure what to do here then

Reply:
```
      const currentPage = this.props.location.pathname
      const nextPage = nextProps.location.pathname
````

Reply:
pathname will always be `/` with our setup i believe

Reply:
fixed